### DISCUSSION TITLE
审核通过未在成员名单中查到

### DISCUSSION THREAD

**Discussion by zcwiki** [travellings-link/travellings#1829] [寻求帮助] [answered]
前几天收到了开往的审核通过通知，但是已经过去几天了，并没有在成员列表中查询到，目前列表还是912个

**Comment by jiangyangcreate** [comment, marked as answer]
> 前几天收到了开往的审核通过通知，但是已经过去几天了，并没有在成员列表中查询到，目前列表还是912个

因为这个机器人只生成数据库插入语句发到我的邮箱，还是需要人授权，为了避免程序不稳定导致数据库紊乱，用一个季度没问题会考虑直接程序写入。时间上，你是在2号周六8：24收到的通知，我在4号周一9：00确认添加。

**Reply by zcwiki** [reply]
感谢解答
